digdag-core/src/test/java/io/digdag/core/workflow/TaskDependenciesTest.java

```diff
@@ -0,0 +1,208 @@
+package io.digdag.core.workflow;
+
+import com.google.common.base.Optional;
+import io.digdag.core.DigdagEmbed;
+import io.digdag.core.LocalSite;
+import io.digdag.core.database.TransactionManager;
+import io.digdag.core.session.ArchivedTask;
+import io.digdag.core.session.StoredSessionAttemptWithSession;
+import java.nio.file.Path;
+import java.util.Collections;
+import java.util.List;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.TemporaryFolder;
+import static io.digdag.core.workflow.WorkflowTestingUtils.loadYamlResource;
+import static io.digdag.core.workflow.WorkflowTestingUtils.runWorkflow;
+import static io.digdag.core.workflow.WorkflowTestingUtils.setupEmbed;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+public class TaskDependenciesTest {
+
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
+
+    @Rule
+    public TemporaryFolder folder = new TemporaryFolder();
+
+    private DigdagEmbed embed;
+    private LocalSite localSite;
+    private Path projectPath;
+    private TransactionManager tm;
+
+    @Before
+    public void setUp()
+            throws Exception {
+        this.embed = setupEmbed();
+        this.localSite = embed.getLocalSite();
+        this.tm = embed.getTransactionManager();
+        this.projectPath = folder.newFolder().toPath();
+    }
+
+    @After
+    public void destroy()
+            throws Exception {
+        embed.close();
+    }
+
+    @Test
+    public void TaskDependenciesTest()
```

Comment:
Do you have any reasons why first letter of the method name is not lowercase? How about `testTaskDependencies`?

Reply:
Oh, I made a mistake...
I wil fix it.